applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -233,12 +225,27 @@ int ElasticIsotropic3D::Check(
 //************************************************************************************
 //************************************************************************************
 
-void ElasticIsotropic3D::CalculateElasticMatrix(
-    Matrix& C,
-    const double E,
-    const double NU
+void ElasticIsotropic3D::CheckClearElasticMatrix(
+    Matrix& C
 )
 {
+    const SizeType size_system = this->GetStrainSize();
+    if (C.size1() != size_system || C.size2() != size_system)
+        C.resize(size_system, size_system, false);
+    C.clear();
```

Comment:
Are you sure this is correct?
Doesn't clear leave the container with size 0?

Reply:
correct. A.clear() sets it to zero but doesn't change the size